game-core/src/main/java/games/strategy/triplea/delegate/MoveDelegate.java

```diff
@@ -189,13 +189,11 @@ public void end() {
       removeAirThatCantLand();
     }
 
-    // WW2V2/WW2V3, fires at end of combat move
-    // WW2V1, fires at end of non combat move
-    if (GameStepPropertiesHelper.isFireRockets(data)) {
-      if (needToDoRockets && TechTracker.hasRocket(bridge.getPlayerId())) {
-        RocketsFireHelper.fireRockets(bridge, bridge.getPlayerId());
-        needToDoRockets = false;
-      }
+    // WW2V1, fires at end of non combat move.
+    // Other versions fire Rockets in BattleDelegate
+    if (needToDoRockets && GameStepPropertiesHelper.isNonCombatMove(data, true)) {
```

Comment:
Based on Panther's overview of rockets, I don't think this is correct. V1 should fire at the same time as all the others.

Reply:
No, it isn't completely correct. I was intending that changing V1 rockets would be out of scope for this PR.

Getting V1 rockets working completely correctly is a huge amount of work! It requires allowing an AA Gun to be nominated as a rocket, moving on Combat Move but only to friendly territories, perhaps some other things but that's enough to be in my too hard basket.